harvest/broker/_base.py

```diff
@@ -24,13 +25,27 @@ class BaseBroker:
         This should be initialized in setup_run (see below).
     """
     
-    def __init__(self, path: str=None):
+    def __init__(self, path: str=None, mode='both'):
```

Comment:
In the future I want better definitions between a streamer and a broker since their main functions will be different.